mesonbuild/compilers/cuda.py

```diff
@@ -213,7 +219,7 @@ def build_rpath_args(self, env: 'Environment', build_dir: str, from_dir: str,
                          rpath_paths: str, build_rpath: str,
                          install_rpath: str) -> typing.List[str]:
         return self._cook_link_args(super().build_rpath_args(
-            build_dir, from_dir, rpath_paths, build_rpath, install_rpath))
```

Comment:
@dcbaker There was one other important change to this line - I switched out `super().` for `self.linker.`

Reply:
Did we actually need to replace the super() call, or does adding the missing `env` variable fix it for you? I'd prefer not to override the super() call to do exactly what the super call is supposed to do.

Reply:
@dcbaker Just the `env` var seems to work, **but** you have accidentally introduced a small typo:

`get_std_exe_link_arg` -> `get_std_exe_link_args`

Reply:
oops, good catch. Updated to fix that.